apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/modules/components/Claimable.tsx

```diff
@@ -82,29 +79,25 @@ function ClaimableModule(props: ModuleInstanceProps) {
   );
 
   const claimConditionQuery = useReadContract(
```

Comment:
Updating it so that if no currency is set, then it will default to the native currency.

Reason being is that in `setClaimCondition.ts`, if the currency is undefined, then it defaults to the native token address